client/src/plugins/libparsec/definitions.ts

```diff
@@ -1361,6 +1373,10 @@ export interface LibParsecPlugin {
         client: Handle,
         token: InvitationToken
     ): Promise<Result<null, ClientCancelInvitationError>>
+    clientChangeAccess(
+        current_access: DeviceAccessStrategy,
+        new_access: DeviceAccessStrategy
```

Comment:
Shouldn't this be a `DeviceSaveStrategy` instead? The `keyFile` shouldn't change.

Reply:
Done